ClinGen/clincoded
# Connects to #472. Added warning message on new family/individual curation pages.

This PR is to add warning message on new family/individual curation pages, as well as to fix invisible individual label input field per Min's commit #3449215.

**Testing steps for #472**
1. Go to the GDM page and add a new PMID.
2. From the far-right column under "Evidence for PMID XXX", add a group curation by clicking on the blue Group bar.
3. Fill out the group curation form and add a family curation to the newly created group curation. And add an individual curation to the added family curation.
4. Upon completion, go back to the GDM page. Now click on the blue Family bar to add a new family curation, and on the curation page, you should expect to see a warning message above the "Family Label" input field stating "If this Family is a member of a Group, please curate the Group first and then add the Family to that Group."
5. Return to the GDM page and click on the blue Individual bar to add a new individual curation. On the curation page, you should expect to see a warning message above the "Individual Label" input field stating "If this Individual is part of a Family or a Group, please curate that Group or Family first and then add the Individual as a member."
6. Return to the GDM page, click on either the "Add new Family to this Group" or "Add new Individual to this Group" links under the Group curation panel, or the "Add new Individual to this Family" link under the Family curation panel. On the subsequent curation pages, you should NOT expect to see any of the warning messages above the input fields.
7. Add a family curation without associating any groups and without attaching any individuals. Upon completing the form, return to the GDM page and click on the "Add new Individual to this Family" link under the family with no associations under the Family curation panel. On the subsequent page, you should expect to see the "Individual Label" input field present below the warning message atop.
